release/src/main/scripts/verify_release_build.sh

```diff
@@ -123,12 +125,14 @@ if [[ ! -z `which hub` ]]; then
   # The version change is needed for Dataflow python batch tests.
   # Without changing to dev version, the dataflow pipeline will fail because of non-existed worker containers.
   # Note that dataflow worker containers should be built after RC has been built.
-  sed -i -e "s/${RELEASE_VER}/${RELEASE_VER}.dev/g" sdks/python/apache_beam/version.py
-  sed -i -e "s/sdk_version=${RELEASE_VER}/sdk_version=${RELEASE_VER}.dev/g" gradle.properties
+  sh "$SCRIPT_DIR"/set_version.sh "$RELEASE_VER"
   git add sdks/python/apache_beam/version.py
   git add gradle.properties
+  # In case the version string was not changed, append a newline to CHANGES.md
+  echo "" >> CHANGES.md
+  git add CHANGES.md
```

Comment:
If it wasn't changed, we don't want to commit it, right? Would `git add --update` work? Or as a last resort there is `git add --ignore-errors` or I guess even just `git add CHANGES.md || echo 'CHANGES.md not modified'`.

Reply:
We still need to submit even the version string wasn't changed in order to test the release branch before tagging RC. That's why I introduced the harmless temporary change in CHANGES.md.